src/Core/Core.csproj

```diff
@@ -70,17 +70,8 @@
     <Compile Include="Enums\HighlighterColor.cs" />
     <Compile Include="Enums\UrlType.cs" />
     <Compile Include="Exceptions\TreeNavigationFailedException.cs" />
-    <Compile Include="Fingerprint\AddResult.cs" />
```

Comment:
>AddResult [](start = 34, length = 9)

The classes being removed are unused outside of the now-defunct FingerprintAction class